chapters/en/slides/module4/module4_03.md

```diff
@@ -0,0 +1,97 @@
+---
+type: slides
+title: 'module4_03'
+---
+
+# Revert your changes
+
+---
+# Revert your changes
+
+
+**reset hard** ✔️
+**revert** ⬅️
+
+Notes: We have already learn how to undo changes deleting our project history with **reset hard**. But what if we want to keep track of the reversion? 
+ There is other way to go back in time that is **revert** your changes.
+
+---
+#  Revert your changes: move to a previous commit (JupyterLab)
```

Comment:
I'd suggest "Go back to" instead of "move to", because reversion implies going back to some previous state and "Go back to" would probably fit the context better here.

```suggestion
#  Revert your changes: Go back to a previous commit (JupyterLab)
```